src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './NewCommentForm.scss';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
+type Props = {
+  handleAddComment: (arg0: string, ar1: string, arg2: string) => void,
```

Comment:
```suggestion
  handleAddComment: (newCommentName: string, newComment: string, newCommentEmail: string) => void,
```
По-суті, це одне і те ж, і існують певні дискусії, який варіант більш правильний, але просто FYI - так писати теж можна і це не є помилкою